docs/content/manual/pre-release/upgrade/auto-upgrade-engine.md

```diff
@@ -13,9 +13,17 @@ title: Automatically Upgrading Longhorn Engine Test
 1. Create a StatefulSet of scale 10 using 10 Longhorn volume. Set node selector so that all pods land on the same node.
 2. Upgrade Longhorn to use a newer default engine image
 3. In Longhorn UI and Longhorn manager logs, Verify that Longhorn doesn't upgrade all volumes at the same time. Only 3 at a time.
-
+4. In longhorn UI, while the concurrent value is set to limit 1, change the value to higher number and verify if the upgrade is happening as per the new value.
+5. Create stateful set pods with volume claim template, set the scale to 10 and in the configure options, select RWO and RWX. Verify the volumes created have RWO set
+6. While the auto upgrade is set, try to manually upgarde the image engine to the latest version.
+7. Take backup for a volume when upgarde hasn't started
+8. Create 10 volmes and set concurrent upgrade limit to 2. When the upgrade starts, delete all the volumes.
+9. Deploy a workload with rwx scale 10 with persistent volume claim template and verify the volume created can be used for all of 10 pods.
+10.While upgrade has started, detach the nodes from the volume and verfiy the detached volume is not upgraded. 
```

Comment:
I thought that detached volume should be upgraded eventually?

Reply:
Upgrade has started - wanted to say that when the upgrade has started, select a node that wasn't already upgraded and try to detach the volume. Therefore I mentioned the detached volume shouldn't be upgraded. I verified this scenario and saw that the volume is upgraded to the latest version as you mentioned. So, is this expected @PhanLe1010 ?